roboflow/models/instance_segmentation.py

```diff
@@ -0,0 +1,72 @@
+import io
+import urllib
+
+import requests
+from PIL import Image
+from requests_toolbelt.multipart.encoder import MultipartEncoder
+
+from roboflow.config import INSTANCE_SEGMENTATION_MODEL, INSTANCE_SEGMENTATION_URL
+from roboflow.util.image_utils import validate_image_path
+from roboflow.util.prediction import PredictionGroup
+
+
+class InstanceSegmentationModel:
+    def __init__(
+        self,
+        api_key,
+        version_id,
+    ):
+        """
+        :param api_key: Your API key (obtained via your workspace API settings page)
+        :param version_id: The ID of the dataset version to use for predicting
+        """
```

Comment:
A couple of the existing model classes seemed to be a bit over-engineered - at least with my current understanding in how these are being used.

I tried to simplify this model as much as possible by making it only using the data that I think it needs.

e.g. The dataset and version can be obtained from the `version_id`, so that is the only argument I require on `init()`. The `local` argument on the other models seemed like a URL string that could be passed in to override the base URL for local/staging development... it seemed a bit cleaner to me to just use `.env` variables for this, so that's the route I went here. If we like this, maybe the others can be similarly refactored.

I also didn't understand the purpose of the `load_model()` method on the other models... nor did I see it used anywhere in the codebase, so I did not add it here.

I nearly made a new `Inference` class that we could make each of these child classes extend from. But that seemed like a rabbit hole best saved for another day. If this is missing anything, please let me know!